js/index.js

```diff
@@ -40,3 +40,102 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Navigation
+let navAnchor = document.querySelectorAll('a');
+navAnchor[0].textContent = siteContent.nav["nav-item-1"];
+navAnchor[1].textContent = siteContent.nav["nav-item-2"];
+navAnchor[2].textContent = siteContent.nav["nav-item-3"];
+navAnchor[3].textContent = siteContent.nav["nav-item-4"];
+navAnchor[4].textContent = siteContent.nav["nav-item-5"];
+navAnchor[5].textContent = siteContent.nav["nav-item-6"];
+
+// Add more elements to the navigation items
+let nav = document.querySelector('nav');
+
+let homeElement = document.createElement('a');
+homeElement.text = 'Home';
+homeElement.href = 'index.html';
+homeElement.style.color = 'green';
+
+let projectElement = document.createElement('a');
+projectElement.text = 'Projects';
+projectElement.href = '#';
+projectElement.style.color = 'green';
+
+nav.prepend(homeElement);
+nav.appendChild(projectElement);
+
+// Make nav anchor links to be color green
+let navAnchorArray = Array.from(navAnchor)
+navAnchorArray.forEach(navItem => navItem.style.color = 'green');
+
+
+
+// Cta
+let cta = document.querySelector('.cta');
+let ctaText = cta.children[0];
+let adjustedCtaText = siteContent.cta["h1"].split(' ').map((word) => {
+  return `${word} <br>`;
+}).join( '');
+ctaText.children[0].insertAdjacentHTML('afterbegin', adjustedCtaText)
+ctaText.children[1].textContent = siteContent.cta["button"];
+cta.children[1].src = siteContent.cta["img-src"];
+
+// Main Content
+let mainContent = document.querySelector('.main-content').children;
+let topContent = mainContent[0];
+let middleContent = mainContent[1];
+let bottomContent = mainContent[2];
+topContent.children[0].children[0].textContent = siteContent["main-content"]["features-h4"];
+topContent.children[0].children[1].textContent = siteContent["main-content"]["features-content"];
+topContent.children[1].children[0].textContent = siteContent["main-content"]["about-h4"];
+topContent.children[1].children[1].textContent = siteContent["main-content"]["about-content"];
```

Comment:
Good job on experimenting with ways to select elements and this is working fine, but I have to say it's really hard to read. 
You could want to find a way to refactor your code in an easier readable way. 
I promise that the future you, others devs and especially me (X-D) will be very glad if you provide clean code.